core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -104,7 +104,9 @@
   /**
    * Adds interceptors that will be called before the channel performs its real work. This is
    * functionally equivalent to using {@link ClientInterceptors#intercept(Channel, List)}, but while
-   * still having access to the original {@code ManagedChannel}.
+   * still having access to the original {@code ManagedChannel}. Interceptors run in the reverse
```

Comment:
not  `{@link }` ? :P

Reply:
It is linked earlier in the same paragraph, so no.